templates/base/vitest.config.ts

```diff
@@ -0,0 +1,31 @@
+import { defineWorkersConfig } from "@cloudflare/vitest-pool-workers/config";
+
+export default defineWorkersConfig(async () => {
+  return {
+    test: {
+      setupFiles: ["./tests/setup.ts"],
+      globals: true,
+      poolOptions: {
+        workers: {
+          wrangler: { configPath: "./wrangler.toml" },
+            miniflare: {
+              compatibilityFlags: ['nodejs_compat'],
+              compatibilityDate: '2024-04-01',
+            }
+        },
+      },
+    },
+    build: {
+      rollupOptions: {
+        external: ["@fiberplane/hono"],
+      }
+    },
+    ssr: {
+      noExternal: [
+        "@sagold/json-pointer",
+        "json-schema-library",
+        "@fiberplane/hono"
+      ],
+    }
```

Comment:
nit - adding a comment about this funkiness so there's some context as to why it's here